polling_stations/apps/file_uploads/models.py

```diff
@@ -56,7 +56,29 @@ def status(self):
         for f in self.file_set.all():
             if not f.csv_valid:
```

Comment:
I think if there's a single file in the fileset, and the ems is Democracy Counts, then the lambda handler will have set `csv_valid` to `False`  and added `"Expected 2 files, found 1"` to the `errors` field on the existing file.
So I think you could just look at `f.errors` here:
```python
for f in self.file_set.all():
    if ("Expected 2 files, found 1" in f.errors) and (self.timestamp + timedelta(seconds=180) > now()):
        return "Waiting"
    elif ("Expected 2 files, found 1" in f.errors) and (self.timestamp + timedelta(seconds=180) < now()):
        return "Error One File"
    
    if not f.csv_valid:
        return "Error"
return "OK"
```

Reply:
Fixed with https://github.com/DemocracyClub/UK-Polling-Stations/pull/4955/commits/10c545be345bbd157df9c4b8ce7c08db57f08d96